README.md

```diff
@@ -1144,6 +1144,91 @@ Please, note that in the above example a preconfigured profile has been used. If
     --project-name $PROJEC_NAME
 ```
 
+### Procurement image management
+
+CloudOS supports procurement functionality to manage and list images associated with organizations within a given procurement. This feature is useful for administrators and users who need to view available container images across different organizations in their procurement.
+
+#### List procurement images
+
+You can get a list of images associated with organizations of a given procurement using the `cloudos procurement images ls` command. This command provides paginated results showing image configurations and metadata.
+
+To list images for a specific procurement, use the following command:
+
+```bash
+cloudos procurement images ls \
+    --cloudos-url $CLOUDOS \
+    --apikey $MY_API_KEY \
+    --procurement-id "your_procurement_id_here"
+```
+
+#### Command options
+
+The `cloudos procurement images ls` command supports the following options:
+
+- `--apikey` / `-k`: Your CloudOS API key (required)
+- `--cloudos-url` / `-c`: The CloudOS URL you are trying to access (default: https://cloudos.lifebit.ai)
+- `--procurement-id`: The specific CloudOS procurement ID (required)
+- `--page`: The response page number (default: 1)
+- `--limit`: The page size limit (default: 10)
+- `--disable-ssl-verification`: Disable SSL certificate verification
+- `--ssl-cert`: Path to your SSL certificate file
+- `--profile`: Profile to use from the config file
+
+#### Example usage
+
+Here's a complete example of how to list procurement images:
+
+```bash
+# Set up your environment variables
+MY_API_KEY="xxxxx"
+CLOUDOS="https://cloudos.lifebit.ai"
+PROCUREMENT_ID="your_procurement_id"
+
+# List images for the procurement (first page, 10 items)
+cloudos procurement images ls \
+    --cloudos-url $CLOUDOS \
+    --apikey $MY_API_KEY \
+    --procurement-id $PROCUREMENT_ID
+```
+
+To get more results per page or navigate to different pages:
+
+```bash
+# Get 25 images from page 2
+cloudos procurement images ls \
+    --cloudos-url $CLOUDOS \
+    --apikey $MY_API_KEY \
+    --procurement-id $PROCUREMENT_ID \
+    --page 2 \
+    --limit 25
+```
+
+#### Using profiles
+
+You can also use profiles to simplify the command. First, configure a profile with your procurement details:
+
+```bash
+cloudos configure set-profile procurement-profile \
+    --apikey $MY_API_KEY \
+    --cloudos-url $CLOUDOS \
+    --procurement-id $PROCUREMENT_ID
+```
+
+Then use the profile:
+
+```bash
+cloudos procurement images ls --profile procurement-profile
+```
```

Comment:
```suggestion
```
I'd suggest to get rid of all this section. It is essentially incorrect (`cloudos configure set-profile` is not a command, it should be `cloudos configure --profile procurement-profile` and takes no argument). If you want, you could add a small entry informing the user `--procurement-id` is now a configurable parameter

Reply:
Added just a small section saying that procurement-id can be configured via profile.